src/addrman.h

```diff
@@ -104,19 +104,22 @@ class CAddrInfo : public CAddress
  *  * Make sure no (localized) attacker can fill the entire table with his nodes/addresses.
  *
  * To that end:
- *  * Addresses are organized into buckets.
- *    * Addresses that have not yet been tried go into 1024 "new" buckets.
+ *  * Addresses are organized into buckets that can each store up to 64 entries.
+ *    * Addresses to which our node has not successfully connected go into 1024 "new" buckets.
  *      * Based on the address range (/16 for IPv4) of the source of information, 64 buckets are selected at random.
```

Comment:
Perhaps mention AS bucketing.
```suggestion
 *      * Based on the address range (/16 for IPv4) of the source of information, or if an asmap is provided, the AS it belongs to (for IPv4/IPv6), 64 buckets are selected at random.
```

Reply:
done as suggested, thanks!